.github/workflows/release.yml

```
@@ -112,6 +114,70 @@
+            fi
+          else
+            echo "⚠️ NORTHFLANK_DEPLOY_HOOK not configured, skipping Northflank deployment"
+          fi
+          
+          echo "🎉 All deployments triggered successfully!"
+
+      - name: 🔍 Verify deployment
+        run: |
+          echo "⏳ Waiting for deployments to complete..."
+          sleep 60
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)